packages/docs/celo-codebase/protocol/plumo.md

```diff
@@ -71,25 +71,28 @@ The instructions below are for the first phase of the Plumo setup, Powers of Tau
 
 ### Generate your address
 
-The first step to participating is generating your Celo address.
+The first step to participating is generating your Celo address in a secure location.
 
 * Download the `generate` binary corresponding to your OS from [here](https://github.com/celo-org/snark-setup-operator/releases/tag/v1.0.0-benchmarking.4).
 * Run it in a command line - navigate in the command line to the relevant folder - and follow the instructions.
     * When asked to `Enter some entropy for your Plumo seed:`, you can use any source of entropy.
     * Make sure to save your passphrase - you will need it later.
-* Send the address generated to kobi@clabs.co. Keep the resulting `plumo.keys` file.
+* Send the address generated to kobi@clabs.co.
```

Comment:
should we create a special plumo@clabs.co that forwards?

Reply:
I think that would be awesome. Is it you who can create or should we ask someone else?

Reply:
hmm its not me. the security team might know who does it